.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
       - run: |
-          make lint cover all
```

Comment:
`all` task in Makefile may be erased?

Reply:
Sure, but I think `make all` is useful to run all tasks locally...